assets/src/dashboard/app/views/editorSettings/components.js

```diff
@@ -138,19 +149,14 @@ export const GridItemContainer = styled.div`
 export const GridItemButton = styled.button`
   display: block;
   background-color: transparent;
-  border: ${({ theme }) => theme.DEPRECATED_THEME.borders.transparent};
+  border: 1px solid transparent;
   width: 100%;
   height: 100%;
   border-radius: 4px;
   border-width: 2px;
   padding: 0;
 
-  ${KEYBOARD_USER_SELECTOR} &:focus {
-    border-color: ${({ theme }) =>
-      rgba(theme.DEPRECATED_THEME.colors.bluePrimary, 0.85)};
-    border-width: 2px;
-    outline: none;
-  }
+  ${({ theme }) => focusableOutlineCSS(theme.colors.border.focus)};
```

Comment:
These focus styles don't show at all, since this button doesn't get focused. I can keep tinkering to make it available for focus though, since it should be for accessibility.

![focus-thing](https://user-images.githubusercontent.com/22185279/106925861-b707c280-66cd-11eb-9606-d22190d77ff5.gif)